Will not be fixed in upstream, we have to keep it locally until our SQLite has
column metadata support

https://rt.cpan.org/Public/Bug/Display.html?id=111942

--- DBD-SQLite-1.48/Makefile.PL	2016-02-12 10:09:44.833772992 +0100
+++ DBD-SQLite-1.48/Makefile.PL	2016-02-12 10:09:38.619446084 +0100
@@ -223,7 +223,6 @@ my @CC_DEFINE = (
 	'-DSQLITE_ENABLE_FTS4',             # for sqlite >= 3.7.4
 	'-DSQLITE_ENABLE_FTS3_PARENTHESIS', # for sqlite >= 3.6.10
 	'-DSQLITE_ENABLE_RTREE',            # for sqlite >= 3.6.10
-	'-DSQLITE_ENABLE_COLUMN_METADATA',
 	'-DSQLITE_ENABLE_STAT3',            # for sqlite >= 3.7.9
 	'-DSQLITE_ENABLE_STAT4',            # for sqlite >= 3.8.3.1
 	'-DNDEBUG=1',
